lib/foreman/renderer/scope/macros/host_template.rb

```diff
@@ -42,6 +42,19 @@ def host_param_false?(name)
             host.params.has_key?(name) && Foreman::Cast.to_bool(host.params[name]) == false
           end
 
+          def root_pass
+            check_host
+            host.root_pass
+          end
+
+          def grub_pass
+            ''
```

Comment:
So based on https://community.theforeman.org/t/grub-pass-ks-console/10610/3 should we add a deprecation warning before we entirely drop the content? if not and the removal is suggested, I'd first drop these macros from community-templates repo, sync them and the remove them completely.